src/nvim/CMakeLists.txt

```diff
@@ -204,14 +206,19 @@ add_custom_command(OUTPUT ${GENERATED_UNICODE_TABLES}
 )
 
 add_custom_command(OUTPUT ${GENERATED_API_DISPATCH} ${API_METADATA}
-    COMMAND ${LUA_PRG} ${DISPATCH_GENERATOR} ${CMAKE_CURRENT_LIST_DIR} ${API_HEADERS} ${GENERATED_API_DISPATCH} ${API_METADATA}
+  COMMAND ${LUA_PRG} ${DISPATCH_GENERATOR} ${CMAKE_CURRENT_LIST_DIR}/api ${API_HEADERS} ${GENERATED_API_DISPATCH} ${API_METADATA}
```

Comment:
it already knows the leaf filename, why not the entire path in the source directory? Someone reading `gendispatch.lua` (with the other scripts) would like to know where the file is, this is easier if it is consistent with other scripts and use `nvimsrcdir`, instead of `libdir` which doesn't say much. Also if some script later wants to use `.lua` files in several sub-directories the existing pattern is simpler.